pkg/activator/util/transports.go

```diff
@@ -49,9 +49,14 @@ var AutoTransport = NewHTTPTransport(http.DefaultTransport, h2cutil.DefaultTrans
 type RetryCond func(*http.Response) bool
 
 // RetryStatus will filter responses matching `status`
-func RetryStatus(status int) RetryCond {
+func RetryStatus(status ...int) RetryCond {
```

Comment:
This functionality is duplicated in the `RetryRoundTripper`, since it supports multiple `RetryCond`s